src/stylesheets/abstracts/_variables.scss

```diff
@@ -1 +1,20 @@
-$color:#E5F4F2;
\ No newline at end of file
+//color
+$color:#E5F4F2;
+$primary-color : #F8D57E;
+$secondary-color : #F8F9FF;
+$text-color-primary:#2D2D2D;
+$text-color-secondary:#009379;
+$button-color-primary : #009379;
+$button-color-secondary:#FFF;
+
+
+//responsiev
+$mobile:429px;
+$medium: 768px;
+$desktop: 1200px;
+$tablet: calc($desktop - 1px);
+
+//font
+$primary-font : PoppinsRegular;
+$secondary-font : PoppinsBold;
+$third-font : MulishRegular;
```

Comment:
?

Reply:
> ?

I saw in my design have two fonts: Mulish and Poppins, but Poppins font not enough to work, so i much use PoppinsRegular and PoppinsBold